release-notes/8.0/preview/api-diff/preview4/Microsoft.NETCore.App/8.0-preview4_System.md

```diff
@@ -0,0 +1,331 @@
+# System
```

Comment:
`System.Runtime` changes LGTM.

CC. @dotnet/area-system-runtime

Reply:
The serialization diffs on the various APIs have the same question as https://github.com/dotnet/core/pull/8469/files#r1198216476 on if anything changed in the tooling here.

Reply:
The main issue was the inability of the tool to generate the diff for new attributes, particularly the obsolete attribute. It should be fixed now.